CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [XOSTOR/create] In warning message of the summary section, take into consideration hosts without disks (PR [#7572](https://github.com/vatesfr/xen-orchestra/pull/7572))
```

Comment:
I'm not sure this message can be easily understood by users.

Reply:
Same for the commit message: what does *take into consideration* mean functionally?